docs/guides/extensions/interfaces-relational-dropdown.md

```diff
@@ -0,0 +1,651 @@
+---
+description: Learn how to create a custom searchable dropdown with items from another collection.
+contributors: Tim Butterfield, Kevin Lewis
+---
+
+# Create A Searchable Dropdown With Items From Another Collection
+
+Interfaces provide a meaningful way for users to provide data. This guide will show you how to create a dropdown menu
+that is searchable for data from another collection.
+
+![A dropdown showing a number of values](https://marketing.directus.app/assets/46b4070d-fd77-4ee3-b08b-654519dda031)
```

Comment:
@phazonoverload the marketing.directus.app urls are note reachable (at least for me)

Reply:
Fixed :)